notebooks/official/pipelines/google_cloud_pipeline_components_model_upload_predict_evaluate.ipynb

```diff
@@ -61,7 +61,7 @@
       "source": [
```

Comment:
<p>nit: remove the extra space</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1922/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>